src/test/java/com/mathworks/ci/RunMatlabBuildBuilderTest.java

```diff
@@ -203,7 +203,7 @@ public void verifyBuildPicksTheCorrectBuildBatch() throws Exception {
         project.getBuildersList().add(this.scriptBuilder);
         FreeStyleBuild build = project.scheduleBuild2(0).get();
         jenkins.assertLogContains("Generating MATLAB script with content", build);
-        jenkins.assertLogContains("buildtool compile", build);
+        jenkins.assertLogContains("compile", build);
```

Comment:
Why doesn't the script contain `buildtool` anymore?

Reply:
It was removed when the script was used. Thanks for pointing it out